docs/guides/color.md

```diff
@@ -0,0 +1,320 @@
+developers who want to include color as part of something they are building.
+
+
+## Colors
+
+In many places, napari allows the specification of color.
+This might be the color of an individual point, the color of text annotations, or a color value in a colormap.
+For example, the face color of all points in a layer can be defined as red.
+
+```python
+In [1]: points.face_color = 'red'
+```
+
+### Standard form
+
+Internally, napari typically coerces an input color value like `'red'` to an RGBA (red, green, blue, alpha)
+array with `shape=(4,)` and `dtype=np.float32` where the values are in `[0, 1]`.
+Each element of this array defines the contribution of each RGBA color component to the overall color,
+where a value of 0 means there is no contribution and a value of 1 means that component is saturated.
+
+For example, a purely `'red'` color is coerced to the array `[1, 0, 0, 1]` where the red component is
+saturated, but the green and blue components do not contribute at all.
+The last element (alpha) similarly describes the opacity of the color, where a value of 0 corresponds to
+transparent and a value of 1 corresponds to opaque.
+Most colors are represented as a soft mix of the RGB color components and an opaque alpha value, such as
+[cornflower blue](https://en.wikipedia.org/wiki/Cornflower_blue) which could be represented as `[0.395, 0.585, 0.930, 1]`.
+
+When storing N color values, a similar array of `shape=(N, 4)` is used where each row represents a single color value.
+For example, reading the points layer's red face color gives us such an array
+
+```python
+In [2]: points.face_color
+Out[2]:
+array([[1., 0., 0., 1.],
+       [1., 0., 0., 1.],
+       [1., 0., 0., 1.]])
+```
+
+where in this case there are `N=3` points.
+
+The [`transform_color`](https://github.com/napari/napari/blob/5f96d5d814aad697c367bdadbb1a57750e2114ad/napari/utils/colormaps/standardize_color.py#L33)
+function is responsible for coercing color input values to the standard form and supports a variety of inputs.
+
+```{note}
+The [`ColorType`](https://github.com/napari/napari/blob/43d9b59da89b9a2b97dba3967a7b28526f9aa0ce/napari/utils/colormaps/colormap_utils.py#L17)
+type definition almost describes all the types of acceptable color inputs, except that it includes
+vispy's `Color` and `ColorArray`, which are not accepted by `transform_color`.
+```
+
+
+### Supported single color input
+
+The single face color could have been defined as red in many equivalent ways.
+
+```python
+# CSS3 name
+points.face_color = 'red'
+
+# matplotlib single character name
+points.face_color = 'r'
+
+# RGB or RGBA hex-code
+points.face_color = '#ff0000'
+points.face_color = '#ff0000ff'
+
+# RGB or RGBA tuple, list, or array
+points.face_color = (1, 0, 0)
+points.face_color = (1, 0, 0, 1)
+points.face_color = [1, 0, 0]
+points.face_color = [1, 0, 0, 1]
+points.face_color = np.array([1, 0, 0])
+points.face_color = np.array([1, 0, 0, 1])
```

Comment:
Alternatively, the "no alpha is alpha=1" concept could go here.

Reply:
Yep, I think this is a good place because it's related to user input. I.e. if I don't provide an alpha value explicitly, this should explain what default value it will take.